.github/pull_request_template.md

```diff
@@ -1,13 +1,25 @@
 <!--
 
-Describe in detail the changes you are proposing, and the rationale.
+Describe in detail the changes you are proposing, and the rationale. If practical, describe how to manually test the change; this isn't always possible, but it makes PRs easier and faster to review.
 
 See the contributing guide:
 
 https://github.com/hashicorp/terraform/blob/main/.github/CONTRIBUTING.md
 
 -->
 
+## Reason for change
+
+## Old behavior
+
+## New behavior
+
+## How to verify
+
+### Example config
+
+### Commands
+
 <!--
 
 Link all GitHub issues fixed by this PR, and add references to prior
```

Comment:
```suggestion
```

Remove this plus lines 26 through 31. (GitHub UI is not letting me make that suggestion for whatever reason.)